website/docs/getting-started/setup/instance-configuration/http-proxy.md

```diff
@@ -0,0 +1,34 @@
+---
+description: This page provides steps to configure HTTP/HTTPS proxy on your self-hosted instance.
+---
+
+# Configure Proxy
+
+Appsmith offers support for running behind a forward proxy, which is beneficial when deploying Appsmith on a server behind a corporate firewall, and allows internet access through a proxy server. This page provides instructions for configuring HTTP/HTTPS proxy on your self-hosted Appsmith instance.
+
+## Prerequisites
+
+- A self-hosted Appsmith instance. See the [installation guides](/getting-started/setup/installation-guides) to set up your Appsmith instance.
+
+## Configure HTTP/HTTPS proxy
+
+Follow these steps to configure HTTP/HTTPS proxy for your self-hosted instance:
+
+1. Go to your Appsmith instance configuration file, such as the `docker.env` file for Docker or the `values.yaml` file for Kubernetes.
+2. Use any one of the below pair of environment variables to set proxy:
+
+ a. `HTTP_PROXY` and/or `HTTPS_PROXY` 
```

Comment:
Can we write

a.  `HTTP_PROXY` or `http_proxy` 
b. `HTTPS_PROXY` or `https_proxy`

instead of the other way around

Reply:
No, either the users can use HTTP_PROXY and HTTPS_PROXY or choose to use lower case. Also, they can choose to use either HTTP_PROXY or HTTPS_PROXY or both. @sharat87 - Please confirm this.